dataset_builders/hf/README.md

```diff
@@ -0,0 +1,128 @@
+
+Annotated 731 user comments on Consumer Debt Collection Practices (CDCP) rule by the Consumer Financial Protection Bureau (CFPB) posted on www.regulationroom.org. (p. 1624)
+
+#### Who are the source language producers?
+
+General public participants, implying American citizens.
+
+According to a voluntary user survey that asked the commenters to self-identify themselves, about 64% of the comments came from consumers, 22% from debt collectors, and the remainder from others, such as consumer advocates and counsellor organizations.
+
+### Annotations
+
+#### Annotation process
+
+The annotators annotated the elementary units and support relations defined in the argumentation model proposed by [Park et al. (2015)](https://dl.acm.org/doi/10.1145/2746090.2746118).
+
+Each user comment was annotated by two annotators, who independently determined the types of elementary units and support relations among them using the GATE annotation tool (Cunningham et al., 2011). A third annotator manually resolved the conflicts to produce the final dataset.
+
+Inter-annotator agreement between 2 annotators is measured with Krippendorf’s α with respect to elementary unit type (α=64.8%) and support relations (α=44.1%); IDs of supported elementary units are treated as labels for the supporting elementary units. (p. 1626)
+
+#### Who are the annotators?
+
+\[More Information Needed\]
+
+### Personal and Sensitive Information
+
+\[More Information Needed\]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+Immediate applications include automatically ranking arguments based on their evaluability for a (crude) identification of read-worthy comments and providing real-time feedback to writers, specifying which types of support for which propositions can be added to construct better-formed arguments. (p. 1624)
+
+### Discussion of Biases
+
+About 45% of the elementary units are `VALUE` type. A significant portion, roughly 75%, of support relation annotations are between adjacent elementary units. While commenters certainly tend to provide reasons immediately after the proposition to be supported, it is also easier for annotators to identify support relations in proximity. Thus, support relations in the wild may be not as skewed toward those between adjacent elementary units. (pp. 1626-1627)
+
+### Other Known Limitations
+
+\[More Information Needed\]
+
+## Additional Information
+
+### Dataset Curators
+
+\[More Information Needed\]
+
+### Licensing Information
+
+\[More Information Needed\]
+
+### Citation Information
+
+```
+@inproceedings{park2018corpus,
+  title={A corpus of erulemaking user comments for measuring evaluability of arguments},
+  author={Park, Joonsuk and Cardie, Claire},
+  booktitle={Proceedings of the Eleventh International Conference on Language Resources and Evaluation (LREC 2018)},
+  year={2018}
+}
+```
+
+### Contributions
+
+Thanks to [@github-username](https://github.com/%3Cgithub-username%3E) for adding this dataset.
```

Comment:
put your github username (if you want)

Reply:
added in 083b80f. And should we also add yours?

Reply:
I think it is fine as it is, you did most of the work for this.